curp/src/server/curp_node.rs

```diff
@@ -65,6 +65,9 @@ pub(super) enum CurpError {
     /// Internal error
     #[error("internal error, {0}")]
     Internal(String),
+    /// Transfer the cluster state
+    #[error("transfer cluster state: {0:?}")]
+    Transfer(Cluster),
```

Comment:
IMO, it's not a good choice to place such a complex data type in `CurpError` since it goes against the assumptions made in error refactoring - errors should carry information as concisely as possible.   If we really need a complex data structure, like Cluster, we should obtain the relevant information through another RPC after receiving this error.

Reply:
Fixed